### ISSUE TITLE
Copy direct URL of email to clipboard 

### ISSUE THREAD

**Issue by leejoramo-d51** [leggett/simplify#809]
Here is a feature request. 

I like to copy a direct link to a specific email to my clipboard. I then use this link in my Note Taking/Task Management  applications to quickly access the messages. Currently I use this bookmarklet: 

```js
javascript:(async function(){try{messageId=document.querySelector('div[data-legacy-message-id]').getAttribute('data-legacy-message-id');user=location.pathname.match(/\/u\/(\d)/)[1];await navigator.clipboard.writeText('https://mail.google.com/mail/u/'+user+'/#all/'+messageId)}catch(e){console.error(e)}})()
```

I think this would make a great addition to Simpliy. I would love for this to be in the message dropdown menus or via a keyboard shortcut.

Once again, Thank You for changing Gmail from rather painful to use, to being very pleasant.

**Comment by leejoramo-d51** [comment]
I see this landed in v3.1.18. 

Thank You

**Comment by leggett** [comment]
Heads up -- while the keyboard shortcut is on by default, the preference to show the Copy button is supposed to be off by default and is shown by error. This will be fixed in the next update (I don't think most users need this feature so the visible button is off by default). 

If you want the button to remain visible, enable here: 
https://mail.google.com/mail/?simplify=hideactions&pref=addMsgLink
